### DISCUSSION TITLE
TCC Error: Undefined symbol'_ftelli64' Error

### DISCUSSION THREAD

**Discussion by WaltuhTabsDev** [raysan5/raylib#2484] [Q&A]
![148374200-77503533-76c9-45ae-9bef-51a1d360cb9c](https://user-images.githubusercontent.com/69824094/169371883-fdc56410-c67e-481d-a11a-c05f29a078dd.png)
I have a problem in tcc raylib when i try to compile any projects even if its from the examples file
its show a error says TCC Error: Undefined symbol'_ftelli64' Error and stop
and its only compile the core examples

**Comment by raysan5** [comment]
@Tabby1527 Are you using latest raylib version from github master branch? In any case, a similar issue was solved some months ago: https://github.com/raysan5/raylib/pull/2319/files

**Reply by WaltuhTabsDev** [reply]
I updated to the latest raylib version from github master branch and I still getting the error

**Reply by razordous** [reply]
I also received an error of tcc: error: undefined symbol '_ftelli64' eventhough after followed https://github.com/raysan5/raylib/pull/2319/files.

Is there any solution. I cannot run any of the 3d_camera examples
